src/sagemaker_mxnet_container/training.py

```diff
@@ -13,14 +13,69 @@
 from __future__ import absolute_import
 
 import logging
+import os
+import socket
+import subprocess
+
+from retrying import retry
 
 import sagemaker_containers.beta.framework as framework
 
+ROLES = ['worker', 'scheduler', 'server']
+
 logger = logging.getLogger(__name__)
 
 
+def _scheduler_host(hosts):
```

Comment:
I think venting this method in the default_save module is fine.

Reply:
done